### PR TITLE
Ml scripts run once

### PR DESCRIPTION
**Pull request by erezh31**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

### CODE REVIEW

**Review by lperryp — APPROVED** [review]
Well done

**Comment by erezh31** [comment]
@Itay4 - it's failing on docker validatiotn. It's an old script with old docker contianer - I don't want to change the docker image for this script. can you please hard merge?

**Comment by erezh31** [comment]
@anara123 @glicht I need to hard merge this PR on docker validation -  I don't want to change the docker image it's old scripts and dockers - I just need to change it to "run once"

File: Packs/CommonScripts/pack_metadata.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "Common Scripts",
     "description": "Frequently used scripts pack.",
     "support": "xsoar",
-    "currentVersion": "1.3.26",
+  "currentVersion": "1.3.27",
```
Comment (anara123) [inline Packs/CommonScripts/pack_metadata.json:5]: fix the indent

File: Packs/CommonScripts/ReleaseNotes/1_3_27.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### GetDuplicatesMlv2
+- Maintenance and stability enhancements
```
Comment (anara123) [inline Packs/CommonScripts/ReleaseNotes/1_3_27.md:4]: Add meaningful release notes.
In this case I think we something important which is this script is going to run on separate docker container each time it called.

**Pull request merged**